### PR TITLE
refactor: storage service request payload

### PR DESCRIPTION
**Pull request by ashleythedeveloper**
## What type of PR is this? (check all applicable)

- [ ] 🍕 Feature
- [ ] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [x] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [x] 📦 Chore (Release)
- [ ] ⏩ Revert

## Description

This PR refactors the storage service request payload by changing how the request ID is generated and passed to the storage service. The ID is now generated using a unique identifier (UUID) rather than relying on filename-based paths. This change affects all components and services that interact with the storage service.
- Updated configurations for various mock-app services to use the new payload structure.
- Modified the upload logic in the storage service to ensure compatibility with the new request structure.
Introduced the crypto service to handle encryption, decryption, and hash computation of the payloads, improving stability and modularity.
- Updated test cases and documentation site to reflect the new payload structure.

## Mobile & Desktop Screenshots/Recordings

Screenshot 2024-10-22 at 11 48 04 AM
Screenshot 2024-10-22 at 11 48 40 AM

Encrypted credential
Screenshot 2024-10-22 at 11 49 56 AM

## Added tests?

- [x] 👍 yes
- [ ] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

Added to documentation?

- [ ] 📜 README.md
- [ ] 📕 storybook
- [x] 🦖 Documentation Site
- [ ] 🙅 no documentation needed

### CODE REVIEW

File: packages/services/src/epcisEvents/aggregationEvent.ts [hunk 1]
```diff
@@ -49,7 +48,7 @@ export const processAggregationEvent: IService = async (
     },
   });
 
-  const aggregationVCLink = await uploadData(storage, aggregationVC, `${identifier}/${generateUUID()}`);
+  const aggregationVCLink = await uploadData(storage, aggregationVC, generateUUID());
```
Comment (namhoang1604) [inline packages/services/src/epcisEvents/aggregationEvent.ts:51]: Why do we remove the path?

File: packages/services/src/storage.service.ts [hunk 2]
```diff
@@ -24,16 +24,13 @@ export const storageService: IStorageService = async (config) => {
...
-
+export const uploadData: IUploadData = async (storage, data, id) => {
   const result = await storageService({
     url: storage.url,
     params: {
       ...storage.params,
-      data: payloadData,
-      filename,
+      data,
+      id,
```
Comment (namhoang1604) [inline packages/services/src/storage.service.ts:33]: Can the `id` be a file path?

Comment (ashleythedeveloper) [reply packages/services/src/storage.service.ts:33]: The storage service is expecting `id`.

**Review by namhoang1604 — APPROVED** [review]
LGTM

**Review by ldhyen99 — DISMISSED** [review]

**Review by namhoang1604 — APPROVED** [review]

**Review by huynguyen-hl — APPROVED** [review]

**Pull request merged**